### DISCUSSION TITLE
Most profitable services

### DISCUSSION THREAD

**Discussion by JonasKenke** [engageub/InternetIncome#189] [Q&A]
Hey everyone,
just wanted to ask what are your most profitable services for you and how much you are making with these currently.
Thanks in advance for sharing :)

**Comment by engageub** [comment]
Hello,
PacketStream, PacketShare, ByteLixir, Traffmonetizer  and Mysterium are paying well at the moment. 
Income depends on the number of proxies and location used. 
For 100 Proxies PacketShare pays about $1.5 per day. For US region it pays more compared to other regions. 
For 100 Proxies Traffmonetizer pays about $0.8 per day. 
For 200 Proxies Bytelixir pays between $0.5 to $1.5 . No app is required. You can add the proxies to bytelixir dashboard.
PacketStream Uses high CPU under load and sometimes it pays about $0.5 per day for 10 proxies.
For 10 IPs, Mysterium pays about $0.6 per day but it depends on the traffic. It can be more if someone purchases the node for long term. 

It would be more profitable if you are using free VPS or you get proxies at cheaper price. 
To conclude, the earnings depend on demand in the location and also buyers who use the service. 

Thank you

**Reply by JonasKenke** [reply]
Hey thank you for sharing your insides but so it looks like it is really hard to get profitable with proxies about 1,5$-2$ + energy cost right?

**Reply by engageub** [reply]
Hello,

You can experiment with a direct connection on your home PC. Docker containers operate in an isolated environment, ensuring they do not interfere with the data on your main host, thanks to volume mappings. These applications are primarily bandwidth-sharing and typically do not require significant CPU resources.

Based on the daily income generated by each application, you can determine which ones should be run with proxies. If you wish to use proxies, it is recommended to start with a single proxy and gradually increase the number as needed.

Thank you

**Reply by peercoder** [reply]
Hi,
Earnings are pretty good with Packetstream but in the past I got banned from many packetstream accounts due to using host proxies, now I'm using isp proxies to see how it goes.
Proxy for Mysterium is not supported at the moment due to ongoing issue. When it fixed?
PacketShare is high earnings but i can't found cheap residential proxies provider to profitable
I also trying ByteLixir use import proxies feature

By the way, I don't know how set up Network3 AI, this is docker logs error:
"you can access the dashboard by opening https://account.network3.ai/main?o=xx.xx.xx.xx:8080 in chrome where xx.xx.xx.xx is the accessible ip of this machine
Node at localhost bound successfully: {'s': -1, 'err': 'the node is not found'}"

Please support!
Thank you!

**Reply by engageub** [reply]
Hello,
For PacketStream, you will have to enable logs and delete the containers where the IPs are detected as non-residential.
Else, your account might get blocked due to sending too many requests for hosting proxies.
Mysterium requires UDP support and there is an ongoing issue https://github.com/xjasonlyu/tun2socks/issues/262 .

For Network3, the nodes will automatically be added to your dashboard https://account.network3.ai/main .
Make sure you are signed in to the website. It requires UDP connections in proxies based on the logs from tun2socks.
You can try with direct connection and check your dashboard if the node is added.


Thank you

**Reply by peercoder** [reply]
Thanks for reply,
How enabling UDP in docker container for UDP connections in proxies?
Thanks!

**Reply by engageub** [reply]
Socks5 protocol supports UDP while http,https and socks4 proxies don't. 
You will have to contact your proxy provider to check if they support UDP in Socks5 before purchasing.
However, it is not guaranteed to work due to ongoing issue mentioned above.
With Multi IP and Multi VPN,  UDP usually works fine  without issues.

**Reply by nkciam** [reply]
Hi @engageub How do I add the proxy to Bytelixir if my proxy site requires authentication by IP? Thanks!

**Reply by engageub** [reply]
Hello,
You can use [pproxy](https://github.com/qwj/python-proxy)  to created authenticated proxy on your VPS via the proxy you have.
Please follow the instructions mentioned in the wiki to setup authenticated proxy. 

Thank you

**Reply by nkciam** [reply]
![image](https://github.com/user-attachments/assets/0770ad60-7d13-47b7-a31d-25c6a901cca7)
Faced with this issue. I used authentication by user/pass type.

**Reply by engageub** [reply]
Hello,
If you use pproxy, you will have to use your VPS IP and PORT with username and password.
Each port maps to different proxy you specify in pproxy configuration.
Did you use docker to start pproxy or you used it with the command line.
Make sure to test the proxies with curl command before adding it to bytelixir dashboard. 

Thank you

**Reply by nkciam** [reply]
Working fine with socks5 proxy. Http proxy type have problem. Thanks!

**Reply by peercoder** [reply]
> Hello, You can use [pproxy](https://github.com/qwj/python-proxy) to created authenticated proxy on your VPS via the proxy you have. Please follow the instructions mentioned in the wiki to setup authenticated proxy.
> 
> Thank you

Hi,
I not found the instructions mentioned in the wiki to setup authenticated proxy.
Please support!
Thanks!

**Reply by engageub** [reply]
> > Hello, You can use [pproxy](https://github.com/qwj/python-proxy) to created authenticated proxy on your VPS via the proxy you have. Please follow the instructions mentioned in the wiki to setup authenticated proxy.
> > Thank you
> 
> Hi, I not found the instructions mentioned in the wiki to setup authenticated proxy. Please support! Thanks!

Hello,

You will have to figure it out through the examples mentioned in the wiki.
If you search with keyword `password` in the wiki, you will find example commands.


Thank you

**Reply by peercoder** [reply]
> Hello, PacketStream, PacketShare, ByteLixir, Traffmonetizer and Mysterium are paying well at the moment. Income depends on the number of proxies and location used. For 100 Proxies PacketShare pays about $1.5 per day. For US region it pays more compared to other regions. For 100 Proxies Traffmonetizer pays about $0.8 per day. For 200 Proxies Bytelixir pays between $0.5 to $1.5 . No app is required. You can add the proxies to bytelixir dashboard. PacketStream Uses high CPU under load and sometimes it pays about $0.5 per day for 10 proxies. For 10 IPs, Mysterium pays about $0.6 per day but it depends on the traffic. It can be more if someone purchases the node for long term.
> 
> It would be more profitable if you are using free VPS or you get proxies at cheaper price. To conclude, the earnings depend on demand in the location and also buyers who use the service.
> 
> Thank you

Hi,

I'm experiencing an issue with my recent Traffmonetizer payment. The platform shows it as "complete," but the funds haven't arrived in my specified wallet.

I'm wondering if anyone else has encountered a similar problem recently? Or is this an isolated issue on my end?

Could those who have received their Traffmonetizer payments recently please confirm? This would help determine if there's a wider issue or if I need to investigate further on my side (e.g., checking wallet address, contacting support directly).

Any insights or similar experiences would be greatly appreciated.

Thanks!

**Reply by engageub** [reply]
Hi,
Thanks for posting here. I have the same issue and sent the information to Traffmonetizer Team about this and awaiting reply.

Thank you

**Reply by peercoder** [reply]
> Hi, Thanks for posting here. I have the same issue and sent the information to Traffmonetizer Team about this and awaiting reply.
> 
> Thank you

Hi engageub,

Thanks for sharing your experience! It's interesting to hear that you had the same issue.

Just wanted to update the thread: the payment status on my end has now switched back to "processing," and I'm happy to report that I have received the payment in my wallet.

It seems there might have been a temporary glitch in their payment status updates. Hopefully, the Traffmonetizer Team will be able to clarify what happened.

Thanks again for your input!

**Reply by engageub** [reply]
Hi,
Just looked into my wallet and received the payment and I do see the same status as "processing". 

Thank you

**Reply by lolhaizz01** [reply]
As I understand it, you're running one IP address for all applications, right? And when running 100 IP addresses, I'll earn as you described.

**Reply by engageub** [reply]
Hi,
Your earnings depend on demand in the location and how the applications distribute the traffic.
Some applications distribute the traffic evenly in all locations while some distribute the traffic based on location.


Thank you

**Reply by S1mps0n** [reply]
@engageub , just wanted to ask you if these proxies you mentioned above for earning (For 100 Proxies PacketShare pays about $1.5 per day, For 100 Proxies Traffmonetizer pays about $0.8 per day.) apply to residential ISP proxies or datacenter SOCKS5/HTTPS which are very low priced (under $0.5)?

**Reply by engageub** [reply]
Hi,
The earnings depend on demand in the location.  Sometimes traffmonetizer pays about $3 per day for 100 IPs.
The comment was made in year 2024 and the earnings vary.
The data was for residential IP address .

Thank you

**Comment by hieunguyen8794** [comment]


**Reply by engageub** [reply]
Hi,

It's great to hear that you have access to a large number of proxies and servers. The code is open source, and the script itself does not impose any limitations on the number of containers you can use—any restrictions will depend solely on your server and network capacity.

If you're working with more than 1,000 proxies, please use the [test branch](https://github.com/engageub/InternetIncome/tree/test) of the repository.

Kindly note that applications requiring a browser tend to consume more system resources compared to others.

I do not use Telegram or Discord for communication regarding this project. If you have any questions related to InternetIncome, feel free to reach out here.

Thank you.

**Comment by dartonomarbun** [comment]
what kinda of proxy provider are you using for the numbers you mention? as you know, quality of proxies makes the results very different.

**Reply by engageub** [reply]
Hi,
Its usually Residential Proxies with high demand in the location.
A particular provider cannot be specified as the earnings vary based on demand in the location. 
Thank you
